<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtTtCableCheckerWindow</class>
 <widget class="QMainWindow" name="mdtTtCableCheckerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actSave"/>
    <addaction name="actNew"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actSave"/>
   <addaction name="actNew"/>
   <addaction name="actRemove"/>
   <addaction name="separator"/>
   <addaction name="actSetTest"/>
  </widget>
  <action name="actSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actSetTest">
   <property name="text">
    <string>Set test</string>
   </property>
  </action>
  <action name="actRemove">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
